packages/synchro-charts/src/components/sc-widget-grid/sc-grid-tooltip.tsx

```diff
@@ -79,24 +96,30 @@ export class ScGridTooltip {
                   </div>
                 )}
 
-                {this.alarmPoint && (
-                  <div>
-                    <div class="awsui-util-label">Status:</div>
-                    <div>
-                      <strong style={{ color }}>{this.alarmPoint.y}</strong> since{' '}
-                      {new Date(this.alarmPoint.x).toLocaleString('en-US', {
-                        hour12: true,
-                        minute: 'numeric',
-                        hour: 'numeric',
-                        year: 'numeric',
-                        month: 'numeric',
-                        day: 'numeric',
-                      })}
-                      {this.breachedThreshold && this.breachedThreshold.description && (
-                        <div>({this.breachedThreshold.description})</div>
-                      )}
-                    </div>
-                  </div>
+                {label ? (
+                  <strong style={{ color }}>{this.breachedThreshold?.label?.text}</strong>
+                ) : (
+                  <fragement>
```

Comment:
is there a reason for a `fragment`? I believe it can just be omitted entirely

Reply:
Done. Remove `fragement`.